imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -0,0 +1,153 @@
+class GlowsHistL0:
+    """Data structure for storing GLOWS histogram packet data.
+    Parameters
+    ----------
+    packet : tuple[list]
+        Histogram packet yielded from space_packet_parser.generate_packets
+
+    Attributes
+    ----------
+    L0_KEYS : tuple[str]
+        Data names from the packet
+    data: dict
+        The data from the packet
+
+    Methods
+    -------
+    process_packet(packet)
+        Process the given packet into the data dictionary
+
+    _convert_histogram_data(binary_hist_data)
+        Process the binary histogram data into 8-bit bins, in a list.
```

Comment:
These should get documented automatically because you have the docstrings associated with those methods below I believe.

Reply:
So I should remove the whole `methods` section? I looked up some numpy pydocs guide and it said to include this.